src/main/java/dao/FruitStoreDaoImpl.java

```diff
@@ -0,0 +1,18 @@
+package dao;
+
+import java.util.List;
+import model.FruitTransaction;
+import strategy.StoreOperationStrategy;
+import strategy.handler.OperationHandler;
+
+public class FruitStoreDaoImpl implements FruitStoreDao {

```

Comment:
```suggestion
public class FrutShopServiceImpl implements FruitShopService {
```
I would choose this name